StudyHub/StudyHub.Common/Requests/InviteUsersRequest.cs

```diff
@@ -1,6 +1,6 @@
-﻿namespace StudyHub.Common.DTO.UserInvitation;
+﻿namespace StudyHub.Common.Requests;
 
-public class InviteUsersDTO
+public class InviteUsersRequest
```

Comment:
Why here request, we mapping it in UserInvitationService?

Reply:
Not correct, firstly we have request from the user, than based on what we got creating DTO, and finally mapped it in InvitedUser entity

Reply:
> Not correct, firstly we have request from the user, than based on what we got creating DTO, and finally mapped it in InvitedUser entity

+++